MM-27254 Include the types of supported groups (#4328)
mattermost/docs: source/deployment/ldap-group-sync.rst

2. Under **Team Management**, enable **Sync Group Members**. If **Anyone can join this team** is enabled or if specific email domains are set, they will be disabled by the Sync Group Members feature.

> 3. Add one or more groups to the team. If there are groups already associated to default users into the team, they will already be present.

Reviewer: ```suggestion
3. Add one or more groups to the team. If there are groups already associated with default users in the team, they'll be listed.
```

2. Under **Channel Management**, enable **Sync Group Members**. Please ensure the channel is set to ``private``.

> 3. Add one or more groups to the channel. If there are groups already associated to default users into the channel, they will already be present.

Reviewer: ```suggestion
3. Add one or more groups to the channel. If there are groups already associated with default users in the team, they'll be listed.
```

> 4. Choose **Add Group** to add one or more groups to the team. If there are groups already associated to default users into the team, they will already be present.

Reviewer: ```suggestion
4. Choose **Add Group** to add one or more groups to the team. If there are groups already associated with default users in the team, they'll be listed.
```

Frequently Asked Questions
----------------------------

Why do my LDAP users and groups exist in Mattermost, but my groups have no members?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> In order for Mattermost to detect group membership correctly, and to automatically add users to the group configured in the **System Console**, you must use one of the following AD/LDAP attributes to represent group members in Mattermost: ``member`` or ``uniqueMember``. These attributes use a ``Distinguished Name`` as the value on groups.

Reviewer: ```suggestion
In order for Mattermost to detect group membership correctly, and to automatically add users to the group configured in the System Console, you must use one of the following AD/LDAP attributes to represent group members in Mattermost: ``member`` or ``uniqueMember``. These attributes use a ``Distinguished Name`` as the value on groups.
```
